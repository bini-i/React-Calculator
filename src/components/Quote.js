import React from "react";

const Quote = () => (
  <p>
    Mathematics is not about numbers, equations, computations, or algorithms: it
    is about understanding. â€“William Paul ThurstonHomeCalculatorQuote
  </p>
);

export default Quote;
