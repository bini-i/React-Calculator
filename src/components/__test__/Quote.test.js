/* eslint-disable no-undef */
import React from "react";
import { render } from "@testing-library/react";
import renderer from "react-test-renderer";
import Quote from "../Quote";

describe("Quote", () => {
  test("renders Quote component", () => {
    render(<Quote />);
  });
});

describe("snapshot test for Quote component", () => {
  test("renders correctly", () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
