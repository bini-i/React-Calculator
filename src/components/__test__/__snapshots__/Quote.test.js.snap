// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test for Quote component renders correctly 1`] = `
<p>
  Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul ThurstonHomeCalculatorQuote
</p>
`;
